src/ui/NoNameFacet/NoNameFacetValues/NoNameFacetValueElement.ts

```diff
@@ -0,0 +1,109 @@
+import { $$ } from '../../../utils/Dom';
+import { NoNameFacet } from '../NoNameFacet';
+import { NoNameFacetValueCheckbox } from './NoNameFacetValueCheckbox';
+import { NoNameFacetValue } from './NoNameFacetValue';
+import { NoNameFacetValueUtils } from './NoNameFacetValueUtils';
+import { l } from '../../../strings/Strings';
+import { KeyboardUtils, KEYBOARD } from '../../../utils/KeyboardUtils';
+
+export class NoNameFacetValueElement {
+  public element: HTMLElement;
+  private labelElement: HTMLElement;
+  private labelWrapperElement: HTMLElement;
+  private captionElement: HTMLElement;
+  private countElement: HTMLElement;
+  private checkboxElement: HTMLElement;
```

Comment:
I'd explode all those parts in different class/stateless function.

Some are extracted (eg: checkbox), while others are not (eg: label)

Reply:
I don't think it's necessary for all those elements, most of them are divs